docs/source/gcp/gettingstarted.rst

```diff
@@ -119,8 +118,9 @@ Next, run the following command to execute the policy with Custodian:
 
 .. code-block:: bash
 
-    custodian run --output-dir=. custodian.yml
+    GOOGLE_CLOUD_PROJECT="project-id" custodian run --output-dir=. custodian.yml
 
+Note: If you already have Google Cloud Project configured for gcloud or Cloud SDK, then you may omit providing it on the command line.
```

Comment:
we don't actually source the gcloud config settings (config file), we just source the same env vars.

Reply:
I was wondering about this - I set the `gcloud` default project but it wasn't picked up. I instead mirrored what was in the AWS docs to suggest it's possible but I wasn't able to figure out

Reply:
in aws identity creds identify target, and the sdks are aligned to the cli, such that the sdks automatically respect cli profile configurations. google instead produces software according to conway's law. the cli team doesn't use github, invented its own api client thats different then the published sdks (of which there are three for python) and doesn't respect sdk configuration, instead does its own thing idiosyncratic thing, which the sdk team then ignores. dysfunctional synergy google style.

nutshell we shouldn't document this as using the gcloud config, we should explicitly recommend setting the environment variable. as an application, we do support all the various environment variables (historical sdk, current sdk, cli) to try and provide some sanity to a user.

Reply:
Sounds good, I've updated the PR to remove this note and continue using the currently suggested CLI/SDK environment variable